rsocket-test/src/commonMain/kotlin/io/rsocket/kotlin/test/TransportTest.kt

```diff
@@ -152,7 +152,7 @@ abstract class TransportTest : SuspendTest {
     }
 
     @Test
-    fun requestResponse100000() = test(TransportTestLongDuration) {
+    fun requestResponse100000() = test(10.minutes) {
```

Comment:
I suspect we want these comfortably inside 1 minute for reliable CI runs.

Reply:
K/N is just not so fast enough :(